ports/stm32/boards/NUCLEO_L432KC/mpconfigboard.h

```diff
@@ -0,0 +1,59 @@
+#define MICROPY_HW_BOARD_NAME       "NUCLEO-L432KC"
+#define MICROPY_HW_MCU_NAME         "STM32L432KC"
+
+#define MICROPY_EMIT_THUMB          (0)
+#define MICROPY_EMIT_INLINE_THUMB   (0)
+#define MICROPY_PY_USOCKET          (0)
+#define MICROPY_PY_NETWORK          (0)
+#define MICROPY_PY_STM              (0)
+#define MICROPY_PY_PYB_LEGACY       (0)
+#define MICROPY_VFS_FAT             (1)
```

Comment:
I think it's best to leave this disabled, to keep this board minimal, then users can enable any additional things they want/need.

Reply:
Ok, will disable VFS_FAT.